input/pagecontent/changes.md

```diff
@@ -12,6 +12,7 @@ This change log documents the significant updates and resolutions implemented fr
 - Removed the requirement for including an offset in the Patient 'birthdate' search parameter [FHIR-47150](https://jira.hl7.org/browse/FHIR-47150)
 - Updated requirement for AU Core Requester to mandate both system and code values for identifier search parameters on Location, Organization, Practitioner and PractitionerRole [FHIR-46782](https://jira.hl7.org/browse/FHIR-46782)
 - Changed the Location search parameter 'address' from SHALL to SHOULD [FHIR-47107](https://jira.hl7.org/browse/FHIR-47107)
+- Added a PractitionerRole search parameter 'role' as a SHOULD [FHIR-47115](https://jira.hl7.org/browse/FHIR-47115)
```

Comment:
Suggest formatting as per suggestion in https://github.com/hl7au/au-fhir-core/pull/281 "Added a PractitionerRole 'role' search parameter...."

Reply:
Done